src/widgets/Rectangle.js

```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types';
 import { DANGER_RED } from '../globalStyles';
 
 const SIZES = {
-  large: { width: 140, height: 40 },
+  large: { width: 120, height: 40 },
```

Comment:
I'm getting this on the emulator:

![Screen Shot 2021-02-17 at 9 07 21 AM](https://user-images.githubusercontent.com/9192912/108115881-a0b32b00-70ff-11eb-9315-43c1544d4da7.png)

maybe the font size needs to come down a bit too?

Reply:
taking it down to `20` is fine